bigquery/tests/scrub_datasets.py

```diff
@@ -0,0 +1,26 @@
+import re
+import sys
+
+from google.api_core.exceptions import NotFound
+from google.cloud.bigquery import Client
+
+
+def main(prefixes):
+    client = Client()
+
+    pattern = re.compile(
+        '|'.join('^{}.*$'.format(prefix) for prefix in prefixes))
+
+    ds_items = list(client.list_datasets())
```

Comment:
I've actually had a lot of trouble with `list_datasets()` in pandas-gbq. This can be quite flakey.

Reply:
Cool. Yeah, such a script is definitely necessary. I ran one in the pandas-gbq project recently, too.

Yeah, `list()` before looping should help with the flakiness.